pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,79 +1,56 @@
-      <span class="md:text-lg">Transaction ID: {{ .ID }}</span>
-    </div>
-    <div class="flex items-center gap-x-2">
-      {{ if eq .Source "remote "}}
-      <button class="p-1 rounded w-32 bg-[#55ACD8] font-semibold text-white md:p-2 hover:bg-[#55ACD8]/80">Edit</button>
-      <button class="p-1 rounded w-32 bg-green-700 font-semibold text-white md:p-2 hover:bg-green-700/80">Accept</button>
-      {{ end }}
-      <a href="/transactions" class="p-1 rounded w-32 bg-black font-semibold text-white text-center md:p-2 hover:bg-black/80">Back</a>
-    </div>
-  </section>
-  <section class="my-8">
-    <div class="mb-2 bg-[#D9D9D9] p-2 rounded">
-      <h2 class="font-bold text-lg">Transaction Details</h2>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Status:</dt>
-      <dd>{{ .Status }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Counterparty:</dt>
-      <dd>{{ .Counterparty }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Counterparty ID:</dt>
-      <dd>{{ .CounterpartyID }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Originator:</dt>
-      <dd>{{ .Originator }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Originator Address:</dt>
-      {{ if eq .OriginatorAddress "" }}
-      <dd class="">&mdash;</dd>
-      {{ else }}
-      <dd>{{ .OriginatorAddress }}</dd>
-      {{ end }}
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Beneficiary:</dt>
-      <dd>{{ .Beneficiary }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Beneficiary Address:</dt>
-      {{ if eq .BeneficiaryAddress "" }}
-      <dd>&mdash;</dd>
-      {{ else }}
-      <dd>{{ .BeneficiaryAddress }}</dd>
-      {{ end }}
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Virtual Asset:</dt>
-      <dd>{{ .VirtualAsset }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Amount:</dt>
-      <dd>{{ .Amount }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Created:</dt>
-      {{ $created := .Created.Format "January 2, 2006 15:04:05" }}
-      <dd>{{ $created }}</dd>
-    </div>
-    <div class="grid grid-cols-2 p-2">
-      <dt class="font-bold">Last Update:</dt>
-      {{ if .LastUpdate.IsZero }}
-      <dd>&mdash;</dd>
-      {{ else }}
-      {{ $lastUpdate := .LastUpdate.Format "January 2, 2006 15:04:05" }}
-      <dd>{{ $lastUpdate }}</dd>
-      {{ end }}
+      <span class="font-semibold">Transaction ID: {{ .ID }}</span>
     </div>
   </section>
-  <!-- TODO: Display ivms101 Data -->
+  <form id="transaction-accept">
+    <section class="py-2 border-t border-t-black ">
+      <h2 class="my-4 font-bold">1. TRANSACTION DETAILS</h2>
+      <input id="id" type="hidden" name="id" value="{{ .ID }}" />
```

Comment:
Though the transaction ID will be visible to users, I wanted to include it in the form as well when sending the request. For now, the source and status are also hidden inputs, but this information can be added if we think it will be helpful.

Reply:
Awesome, thanks! I think that's a good idea to show the transaction ID - since that will be an important link to other compliance activities.

Right now this form is focused on the Transaction data type; but the accept/reject action is about secure envelopes (the transaction is just high level meta data for an encrypted secure envelope). No changes are required for this PR, but I wanted to alert you to the fact that this will become an IVMS101 form at some point (once we get the backend together).